src/main/java/uk/gov/justice/digital/service/DeltaLakeService.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.justice.digital.service;
+
+import io.delta.tables.DeltaTable;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+
+public class DeltaLakeService {
+
+    public boolean exists(final String prefix, final String schema, final String table) {
+        return DeltaTable.isDeltaTable(getTablePath(prefix, schema, table));
+    }
+
+    private String getTablePath(final String prefix, final String schema, final String table) {
```

Comment:
This duplicates code in the Zone base class. The methods there could be factored out into a helper class.